test/models/page.ts

```diff
@@ -222,14 +224,16 @@ describe('Page model', () => {
 
     const savedPage = await page.save();
 
-    const foundPage = await Page.get(savedPage._id);
-
-    const {data} = foundPage;
-
-    expect(data._id).to.equal(savedPage._id);
-    expect(data.title).to.equal(initialData.body.blocks[0].data.text);
-    expect(data.uri).to.equal(transformToUri(initialData.body.blocks[0].data.text));
-    expect(data.body).to.deep.equal(initialData.body);
+    if (savedPage._id !== undefined) {
```

Comment:
Let's invert condition, so code would look neater. If`_id` is undefined return, otherwise everything else